Dockerfile

```diff
@@ -64,8 +65,17 @@ RUN apt update \
     libgtk-3-dev \
     libcanberra-gtk-module \
     graphviz
+
+# Install Qt6
+RUN apt update \
+    && apt install --no-install-recommends -y \
+        qt6-base-dev \
+        libxcb-cursor0 \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Do we also want to find an alternative for Qt C++? Or only for Qt Python?

Reply:
We just need an alternative for PyQt.  Both PyQt and PySide are python bindings for C++ Qt which @AndreasHeumann has filed SWIPAT for in the past https://nvbugswb.nvidia.com/NvBugs5/SWBug.aspx?bugid=4227741 

PyQt is not developed by Qt and so has additional licensing on top of it, so switching to PySide now which is from Qt.

Reply:
Okay, so we will recommend that users use Qt for C++ and PySide for Python apps, and that will satisfy licensing due to both being available under LGPL? That makes sense to me, thanks for clarifying.